docs/content/tutorial/step_09.ngdoc

```diff
@@ -34,16 +34,16 @@ Since we are using [npm][npm] to install client-side dependencies, this step upd
   "name": "angular-phonecat",
   ...
   "dependencies": {
-    "angular": "1.7.x",
-    "angular-route": "1.7.x",
+    "angular": "1.8.x",
+    "angular-route": "1.8.x",
```

Comment:
For these to be valid, we should also update the [actual tutorial][1] :grin:

[1]: https://github.com/angular/angular-phonecat

Reply:
Yeah - a job for someone.... 👀 🤔

Reply:
Ha ha :smiley: Happy to do it once 1.8 is out.